packages/inscription-view/src/components/browser/conditionBuilder/Condition.tsx

```diff
@@ -0,0 +1,113 @@
+import { BasicField, Button, Flex, Label, Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useConditionBuilderContext } from './ConditionBuilderContext';
+import InputWithBrowser from '../../widgets/input/InputWithBrowser';
+
+const typeOptions = [
+  { label: 'equal to', value: '==' },
+  { label: 'not equal to', value: '!=' },
+  { label: 'is true', value: 'isTrue' },
+  { label: 'is false', value: 'isFalse' },
+  { label: 'is empty', value: 'isEmpty' },
+  { label: 'is not empty', value: 'isNotEmpty' },
+  { label: 'less than', value: '<' },
+  { label: 'greater than', value: '>' },
+  { label: 'less or equal to', value: '<=' },
+  { label: 'greater or equal to', value: '>=' }
+];
+type ConditionType = (typeof typeOptions)[number]['value'];
+
+export const logicalOperatorOptions = [
+  { label: 'and', value: '&&' },
+  { label: 'or', value: '||' }
+];
+export type LogicalOperator = (typeof logicalOperatorOptions)[number]['value'];
+
+export interface Condition {
+  argument1: string;
+  operator: ConditionType;
+  argument2: string;
+  logicalOperator: LogicalOperator;
+}
+
+export interface ConditionProps {
+  condition: Condition;
+  conditionIndex: number;
+  groupIndex: number;
+  conditionsCount: number;
+}
+
+export const Condition = ({ condition, conditionIndex, groupIndex, conditionsCount }: ConditionProps) => {
+  const { updateCondition, removeCondition } = useConditionBuilderContext();
+
+  return (
+    <Flex
+      direction='column'
+      style={{ border: 'var(--basic-border', borderRadius: 'var(--border-r2)', padding: 'var(--size-2)' }}
+      className='condition-builder__condition'
+    >
+      <Flex direction='row' justifyContent='space-between'>
+        <Label>{`Condition ${conditionIndex + 1}`}</Label>
+        <Button onClick={() => removeCondition(groupIndex, conditionIndex)} icon={IvyIcons.Trash} aria-label='Remove Condition' />
+      </Flex>
+      <Flex alignItems='center' gap={2} direction='row'>
+        <InputWithBrowser
+          value={condition.argument1}
+          onChange={val => updateCondition(groupIndex, conditionIndex, 'argument1', val)}
+          browsers={['attr']}
```

Comment:
For now, I've only integrated the attribute browser. Would it also make sense to include another type of browser, such as the function or type browser?